OpenSim/Simulation/Model/CMCActuatorSubsystem.cpp

```diff
@@ -146,14 +142,21 @@ void CMCActuatorSubsystemRep::setSpeedTrajectories(FunctionSet *aSet) {
         _qSet->evaluate(_uWork,1,t);
     }
 
-    for(i=0;i<nq;i++) q[i] = _qWork[i] + _qCorrections[i];
-    for(i=0;i<nu;i++) u[i] = _uWork[i] + _uCorrections[i];
+    /* Hack to obtain a modifiable state in a const method */
```

Comment:
The issue is that the CMCActuatorSubsystemRep has it own secret internal State (_completeState) which it updates. Computation is not supposed change the System (or subsystem). It is a bad implementation and I would rather not change it. All I did was gather it to one `const_cast` instead of having three.